scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -386,6 +414,9 @@ main (int argc, char *argv[])
   /* Sensitivity model includes implementation loss and noise figure */
   spectrumWifiPhy.Set ("CcaMode1Threshold", DoubleValue (-79));
   spectrumWifiPhy.Set ("EnergyDetectionThreshold", DoubleValue (-79 + 3));
+  /* Custom error rate model for IEEE 802.11ad */
+  spectrumWifiPhy.SetErrorRateModel ("ns3::DmgErrorModel",
+                                     "FileName", StringValue ("DmgFiles/ErrorModel/LookupTable_1458.txt"));
```

Comment:
This error rate model comes with the QD channel model. And it has been used in all QD evaluation scripts except this one. Therefore I added it. I am not able to say why we should use it since I didn't work on it. But in principle, this error model should be for accurate than sensitivity. I would retain sensitivity as the default one for now.